pages/docs/data-ops/cli/v3/dbt-integration.mdx

```diff
@@ -251,14 +257,12 @@ To import all dbt models in a folder as Hashboard models, simply add
 
 ```yaml filename="dbt_project.yml"
 models:
-  the_models:
+  folder:
```

Comment:
Not new in this PR but is this example correct? Seems like it should be `hashboard:` instead of `hashboard-defaults`?

Reply:
both work it'll get merged in automatically afaik